packages/mattermost-redux/src/selectors/entities/channels.ts

```diff
@@ -666,6 +667,99 @@ export const getUnreadStatus: (state: GlobalState) => BasicUnreadStatus = create
     },
 );
 
+/**
+ * Return a tuple of
+ * - Set of team IDs that have unread messages
+ * - Map with team IDs as keys and unread mentions counts as values
+ */
+export const getTeamsUnreadStatuses: (state: GlobalState) => [Set<Team['id']>, Map<Team['id'], number>] = createSelector(
+    'getTeamsUnreadStatuses',
+    getAllChannels,
+    getMyChannelMemberships,
+    getChannelMessageCounts,
+    getUsers,
+    getCurrentUserId,
+    isCollapsedThreadsEnabled,
+    getThreadCounts,
+    (
+        channels,
+        channelMemberships,
+        channelMessageCounts,
+        users,
+        currentUserId,
+        collapsedThreads,
+        teamthreadCounts,
```

Comment:
Slight nitpicking, but would you mind changing the names of these two? `collapsedThreads` makes it sound a bit too much like an object, so `collapsedThreadsEnabled` or `crtEnabled` would be a bit clearer, and the capitalization of `teamthreadCounts` is incorrect

Reply:
Yes changed